src/Microsoft.AspNet.Mvc/RazorPreCompileModule.cs

```diff
@@ -19,10 +20,15 @@ namespace Microsoft.AspNet.Mvc
     public abstract class RazorPreCompileModule : ICompileModule
     {
         private readonly IServiceProvider _appServices;
+        private readonly IMemoryCache _memoryCache;
 
         public RazorPreCompileModule(IServiceProvider services)
         {
             _appServices = services;
+            // When ListenForMemoryPressure is true, the MemoryCache evicts items at every gen2 collection.
+            // In DTH, gen2 happens frequently enough making it an undesirable characteristic for caching
```

Comment:
nits:
- "making it an undesirable characteristic" -> "to make it undesirable"
- might mention something like "not using `IMemoryCache` from DI for this reason as well as completely different (design time vs. runtime) purposes."